app/helpers/value_groups_helper.rb

```diff
@@ -1,28 +1,40 @@
 module ValueGroupsHelper
   def value_group_pie_data(value_group)
-    values = value_group.children
-      .map do |child|
-        {
-          label: to_accountable_title(Accountable.from_type(child.name)),
-          percent_of_total: child.percent_of_total.round(1).to_f,
-          value: child.sum.amount.to_f,
-          currency: child.sum.currency.iso_code,
-          currency_symbol: child.sum.currency.symbol,
-          formatted_value: format_money(child.sum.amount.to_f, precision: 2, unit: child.sum.currency.symbol, separator: ".", delimiter: ","),
-          bg_color: accountable_bg_class(child.name),
-          fill_color: accountable_fill_class(child.name)
-        }
-      end
-      .filter { |child| child[:value] > 0 }
+    values = value_group.children.map do |child|
+      {
+        label: to_accountable_title(Accountable.from_type(child.name)),
+        percent_of_total: child.percent_of_total.round(1).to_f,
+        value: child.sum.amount.to_f,
+        value_str: {
+          fractional_part: child.sum.cents_str,
+          main_part: format_money_without_symbol(child.sum.amount.to_f, precision: 0, unit: child.sum.currency.symbol)
+        },
+        currency: {
+          iso_code: child.sum.currency.iso_code,
+          symbol: child.sum.currency.symbol,
+          displayed_before_value: child.sum.currency.default_format.start_with?("%u")
+        },
+        bg_color: accountable_bg_class(child.name),
+        fill_color: accountable_fill_class(child.name)
+      }
+    end.filter { |child| child[:value] > 0 }
 
-    total = values.sum { |child| child[:value] }
-    formatted_total = format_money(total, precision: 2, unit: value_group.sum.currency.symbol, separator: ".", delimiter: ",")
+    total_value = values.sum { |child| child[:value] }
+    main_part = format_money_without_symbol(total_value, precision: 0, unit: value_group.sum.currency.symbol)
+    fractional_part = Money.new(total_value, value_group.sum.currency).cents_str
 
     {
       values: values,
-      total: total,
-      formatted_total: formatted_total,
-      currency_symbol: value_group.sum.currency.symbol
+      value: total_value,
+      value_str: {
+        main_part: main_part,
+        fractional_part: fractional_part
```

Comment:
i have choose terms more generic like ``main_part`` and ``fractional_part``, do you think this is good or you want to stick with something like ``dollars_str`` and ``cents_str`` ?